PayPal/src/test/java/com/braintreepayments/api/PayPalPendingRequestUnitTest.kt

```diff
@@ -0,0 +1,64 @@
+package com.braintreepayments.api
+
+import android.net.Uri
+import org.json.JSONException
+import org.json.JSONObject
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class PayPalPendingRequestUnitTest {
+
+    private val browserSwitchRequest = BrowserSwitchRequest(
+        1,
+        Uri.parse("http://"),
+        JSONObject().put("test_key", "test_value"),
+        "return-url-scheme",
+        false
+    )
+
+    private val browserSwitchPendingRequest =
+        BrowserSwitchPendingRequest.Started(browserSwitchRequest)
+
+    private val payPalBrowserSwitchRequest = PayPalBrowserSwitchRequest(browserSwitchPendingRequest)
+
+    @Test
+    fun startedConstructor_fromString_createsBrowserSwitchRequest() {
```

Comment:
could we use backticks around the function name to allow for spaces? it would make the file and reporting more readable.

```suggestion
    fun `started constructor fromString creates BrowserSwitchRequest`() {
```

Reply:
Ah yes good idea - most of our tests are still in Java, so we don't have a naming convention around this. But agreed that backticks and readable method names is helpful and should be used going forward.